django-backend/fixtures/e2e-test-additional-users.json

```diff
@@ -0,0 +1,157 @@
+[

```

Comment:
Actually yes.  This places additional users in the db in order to test the functionality of the new page.  This will prove useful again when it comes time to do the E2E tests for this page.